docs/datasets.md

```diff
@@ -1,3 +1,54 @@
 # Datasets
 
-Place holder until we move the docs from armory/data/README.md
\ No newline at end of file
+The `armory.data.datasets` module implements functionality to return NumPy data 
+generators of various data modalities. Currently our datasets are loaded using 
+TensorFlow Datasets from cached tfrecord files served from the armory S3 buckets.
+
+
+### ArmoryDataSet Generator
+All datasets return an `ArmoryDataGenerator` which implements the methods needed 
+by the ART framework. Specifically `get_batch` will return a tuple of `(data, labels)` 
+for a specified batch size.
+
+
+### Image Datasets
+
+| Dataset    | Description | x_shape | x_dtype  | y_shape  | y_dtype |
+|:----------: |:-----------: |:-------: |:--------: |:--------: |:-------: |
+| cifar10 | CIFAR 10 classes image dataset | (N, 32, 32, 3) | uint8 | (N,) | int64 |
+| german_traffic_sign | German traffic sign dataset | (N, variable_height, variable_width, 3) | uint8 | (N,) | int64 |
+| imagenette | Smaller subset of 10 classes from Imagenet | (N, variable_height, variable_width, 3) | uint8  | (N,) | int64 |
+| mnist | MNIST hand written digit image dataset | (N, 28, 28, 1) | uint8 | (N,) | int64 |
+| resisc45 | REmote Sensing Image Scene Classification | (N, 256, 256, 3) | uint8 | (N,) | int64 |
+| imagenet_adversarial | ILSVRC12 adversarial dataset on ResNet50 | (N, 224, 224, 3) | uint8 | (N,) | int64 |
+
+<br>
+### Audio Datasets
+| Dataset    | Description | x_shape | x_dtype  | y_shape  | y_dtype |
+|:----------: |:-----------: |:-------: |:--------: |:--------: |:-------: |
+| digit | Audio dataset of spoken digits | (N, variable_length) | int64 | (N,) | int64 |
+| librispeech_dev_clean | Librispeech dev dataset for speaker identification  | (N, variable_length)  | int64 | (N,)  | int64 |
+
+<br>
+### Video Datasets
+| Dataset    | Description | x_shape | x_dtype  | y_shape  | y_dtype |
+|:----------: |:-----------: |:-------: |:--------: |:--------: |:-------: |
+| ucf101 | UCF 101 Action Recognition | (N, variable_frames, 240, 320, 3) | uint8 | (N,) | int64 |
```

Comment:
The `<br>` breaks the markdown formatting in GitHub:
https://github.com/twosixlabs/armory/blob/887d43ee5dd38b79945dc2ac3b994f2a989d1d83/docs/datasets.md

Reply:
Thanks. Pushed a fix. The CSS table style is still visible on the markdown version, but it's on the bottom and not a major concern IMO. Let me know what you think.

Reply:
I think that's fine for now.